native_client/swift/deepspeech_ios/DeepSpeech.swift

```diff
@@ -0,0 +1,301 @@
+//
+//  DeepSpeech.swift
+//  deepspeech_ios
+//
+//  Created by Reuben Morais on 14.06.20.
```

Comment:
is this generated? hand-crafted? if it's generated, we should have doc on how to regenerate

Reply:
Hand written.